src/main/java/IPChat.java

```diff
@@ -0,0 +1,41 @@
+import java.util.Scanner;
+public class IPChat {
+    public static void main(String[] args) {
+
+        // Level 3: Mark as done
+        System.out.println("Hello I'm IPChat, What can I do for you");
+        System.out.println("------------------------------------------");
+
+        Scanner input =  new Scanner(System.in);
+        String statements = input.nextLine(); // changed statement to statements
```

Comment:
Java coding standard: Comment should be indented relative to their position in code. 

Java code quality: Comment minimally but sufficiently - Comments should be written to the reader, and not be a note to self. But this is something that will be covered later in the lectures so it is just good to take note.